### PR TITLE
Update and pin the netty version to 4.1.16.Final.

### PR DESCRIPTION
**Pull request by anwar6953**
Update and pin the netty version to 4.1.16.Final.

### CODE REVIEW

File: wrangler-core/src/test/java/co/cask/directives/external/InvokeHttpTest.java [hunk 1]
```diff
@@ -49,16 +48,16 @@
...
 public class InvokeHttpTest {
   private static NettyHttpService httpService;
-  protected static String baseURL;
+  private static String baseURL;
 
   @Before
   public void startService() throws Exception {
     List<HttpHandler> handlers = new ArrayList<>();
     handlers.add(new ServiceHandler());
     httpService = NettyHttpService.builder("Services")
```
Comment (chtyim) [inline wrangler-core/src/test/java/co/cask/directives/external/InvokeHttpTest.java:58]: Should have a direct dependency on the netty-http library.

File: wrangler-core/src/test/java/co/cask/directives/external/InvokeHttpTest.java [hunk 2]
```diff
@@ -76,24 +75,20 @@ public void destroy(HandlerContext handlerContext) {
...
-    public void append(HttpRequest request, HttpResponder responder) {
+    public void append(FullHttpRequest request, HttpResponder responder) {
       Map<String, Object> object = postRequest(request);
-      Map<String, Object> response = new HashMap<>();
       String c = String.format("%s:%f", object.get("a"), object.get("b"));
-      String headerC = request.getHeader("C");
-      response.put("c", c);
-      response.put("HeaderC", headerC);
-      responder.sendJson(HttpResponseStatus.OK, response);
+      String headerC = request.headers().get("C");
```
Comment (chtyim) [inline wrangler-core/src/test/java/co/cask/directives/external/InvokeHttpTest.java]: A better way is to use the `@HeaderParam("C")` annotation

File: wrangler-core/src/test/java/co/cask/directives/external/InvokeHttpTest.java (hunk 1 above)
Comment (anwar6953) [reply wrangler-core/src/test/java/co/cask/directives/external/InvokeHttpTest.java:58]: I think its already there. Just need to update the version...
edit: done

**Review by chtyim — COMMENTED** [review]
LGTM

**Comment by anwar6953** [comment]
Squashed commits. `mvn clean test` passed locally.
The bamboo build won't pass yet, because the cdap artifacts with the latest netty changes haven't been pushed yet to the snapshot repository.

**Review by chtyim — APPROVED** [review]

**Pull request merged**